src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -12,6 +12,7 @@
             [metabase.driver.sql-jdbc.sync.common :as common]
             [metabase.driver.sql-jdbc.sync.interface :as i]
             [metabase.driver.sql.query-processor :as sql.qp]
+            [metabase.mbql.schema :as mbql-schema]
```

Comment:
I think literally everywhere else in the codebase we alias this as `mbql.s`, so for consistency sake you should do the same thing. It makes stuff easier to grep.

@bshepherdson is working on https://github.com/metabase/metabase/issues/19930 to make namespace aliases consistent so depending on which of these lands first it's going to make more work for somebody